### PR TITLE
Update Contact Details

### PR DESCRIPTION
**Pull request by bhuvansingla**
- Update coordinators 2019-20
- Update secretaries 2019-20
- Update ex-coordinators

### CODE REVIEW

File: contact.html [hunk 1]
```diff
@@ -3,27 +3,129 @@
...
-    mail: ayushb@iitk.ac.in
-    github: ayush268
-    image: ayush.jpg
-  'Parv Mor':
-    number: +917318019085
-    mail: parv@iitk.ac.in
-    github: parvmor
-    image: parv.jpg
+  'Aditya Gulati':
+    number: +910006548714
```
Comment (APwhitehat) [inline contact.html]: This number looks weird. @pclubiitk/coordies-2019-20 ?

File: _config.yml [hunk 2]
```diff
@@ -18,6 +18,27 @@ facebook_username:  pclubiitk
...
+      mail: anantp@iitk.ac.in
+      github: apwhitehat
+      image: anant.jpg
+      body:
+    'Ayush Bansal':
+      mail: ayushb@iitk.ac.in
+      github: ayush268
+      image: ayush.jpg
+      body:
+    'Parv Mor':
```
Comment (APwhitehat) [inline _config.yml]: Strictly speaking, Parv was relieved of his responsibilities. We may want to remove him from this list.

**Review by APwhitehat — APPROVED** [review]
Looks good otherwise.

File: _config.yml (hunk 2 above)
Comment (bhuvansingla) [reply _config.yml]: Done

File: contact.html (hunk 1 above)
Comment (bhuvansingla) [reply contact.html]: Updated

**Comment by bhuvansingla** [comment]
Issues raised were resolved. Is this ready to merge now? :)

**Comment by ditttu** [comment]
Could you squash the commits?

**Comment by bhuvansingla** [comment]
Squashed! Thanks for pointing out :)

**Pull request merged**